src/librustc_codegen_llvm/back/lto.rs

```diff
@@ -885,13 +894,19 @@ pub unsafe fn optimize_thin_module(
 pub struct ThinLTOImports {
```

Comment:
This name is not ideal anymore, since it also contains exports.

Same applies to the `{curr,prev}_import_map` variable name in these lines above:

```rust
//...
                let curr_imports = curr_import_map.modules_imported_by(module_name);
                let prev_exports = prev_import_map.modules_exported_by(module_name); 
//...
```

I had to stare for good couple mins at them to notice that these are not just duplicated lines and are _ex_ not _im_.

Reply:
fair enough, let me see what I can do to address that.